recipes/cykdtree/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: cykdtree
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cykdtree/cykdtree-{{ version }}.tar.gz
+  sha256: e58104def3f030dc9bd73e4b9a1575cedafec78354d3d37fa455f3478a3200f2
+  patches:
+    - no_test_import.patch
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - cython
```

Comment:
You need compilers here

Reply:
Not yet. I'll adapt that to the new format in the next release.
(unless people already removed the compiler from the  Dockerimage!?)